src/workspace/qml/MuseScore/Workspace/internal/WorkspacesView.qml

```diff
@@ -51,28 +51,28 @@ RadioButtonGroup {
     }
 
     delegate: RoundedRadioButton {
-        width: ListView.view.width
-        leftPadding: root.leftPadding
+        ButtonGroup.group: root.radioButtonGroup
 
+        width: ListView.view.width
         height: 46
+
+        leftPadding: root.leftPadding
         spacing: 12
 
-        text: model.name
-        font: model.isSelected ? ui.theme.bodyBoldFont : ui.theme.bodyFont
+        text: modelname
```

Comment:
Something went wrong here... Probably you should revert these changes

Reply:
Very strange issue! 
Anyway, thanks!
Fixed.